### ISSUE TITLE
Inserting doors into an island

### ISSUE THREAD

**Issue by TEDSTANDAGE** [CCDevNelson/Snap-Updates#94]
Riya and I have discussed reworking island components, but there are couple items we need to address before we can do that.

Along with Item #93 , another issue exists. I am able to insert drawers, hampers, and shelves into an island, but when I try to insert doors I end up with the following list of errors:

![image](https://user-images.githubusercontent.com/61025894/84822109-054a4300-afd1-11ea-8a80-3b34cbeb49a3.png)
